src/oci.c

```diff
@@ -523,6 +526,90 @@ cc_oci_attach (struct cc_oci_config *config,
 }
 
 /*!
+ * Get the home directory for the workload user
+ *
+ * \param config \ref cc_oci_config.
+ *
+ * \return home dir for user on sucess, "/" by default
+ */
+
+private gchar*
+get_user_home_dir(struct cc_oci_config *config) {
+	gchar          *user_home = NULL;
+	gchar          *password_path = NULL;
+	FILE           *pw_file = NULL;
+	struct passwd  *pw_entry;
+
+	if (! config) {
+		goto end;
+	}
+
+	password_path = g_strdup_printf ("%s/%s", config->oci.root.path, "etc/passwd");
+	pw_file = g_fopen (password_path, "r");
```

Comment:
Thats a good point. I could  fallback to /usr/share/etc/passwd for stateless systems if we are not able to to retrieve the the home dir from /etc/passwd.